settings.py

```diff
@@ -736,6 +736,7 @@
     'HUBSPOT_API_ID': '',
     'GTM_ID': '',
     'DRIFT_ID': '',
+    'APPCUES_ID': '',
```

Comment:
Should this also have `APPCUES_KEY`?

Reply:
...Realized that we don't actually use `APPCUES_KEY` at this point, but for the sake of consistency I'd either add it here or remove it from the code added in https://github.com/dimagi/commcare-cloud/pull/1343/

Reply:
I'll add it here. It seems like it may be useful at some point